plugins/modules/site_workflow_manager.py

```diff
@@ -130,7 +130,7 @@
               units_of_measure:
                 description: |
                     Specifies the unit of measurement for floor dimensions, typically 'feet' or 'meters'.
```

Comment:
Specifies the unit of measurement for floor dimensions, such as 'feet' or 'meters'.  
    This field was introduced in version 2.3.7.6 and is optional.

Reply:
Added
default: feet